compendium-browser.js

```diff
@@ -980,12 +1015,23 @@ class CompendiumBrowser extends Application {
     decorateItem(item5e) {
         if (!item5e) return null;
         //Decorate and then filter a compendium entry - returns null or the item
-        const item = item5e.data;
 
+        //JV-080 - v10 does away with item.data and everything is under #system but we want to decorate the first level of the item for return
+        let item = item5e;
+
+        if (this.isVersion10) {
+            item.level = item5e.system?.level;
+            item.components = item5e.system?.components;
+        }
+        else {
+            item = item5e.data;
+            item.level = item5e.data?.level;
+            item.components = item5e.data?.level;
+        }
         // getting damage types (common to all Items, although some won't have any)
         item.damageTypes = [];
-        if (item.data.damage && item.data.damage.parts.length > 0) {
-            for (let part of item.data.damage.parts) {
+        if (item.system.damage && item.system.damage.parts.length > 0) {

```

Comment:
I might be reading this wrong, but is item.system.damage here within a test for v10? If not, then won't this fail for v0.8? I thought that the switch to item.system was introduced in v10, but I'm quite possibly wrong.

See my top-level comment about setting minimum-compatible version to be v10